schema/pod.go

```diff
@@ -28,13 +28,15 @@ import (
 const PodManifestKind = types.ACKind("PodManifest")
 
 type PodManifest struct {
-	ACVersion   types.SemVer        `json:"acVersion"`
-	ACKind      types.ACKind        `json:"acKind"`
-	Apps        AppList             `json:"apps"`
-	Volumes     []types.Volume      `json:"volumes"`
-	Isolators   []types.Isolator    `json:"isolators"`
-	Annotations types.Annotations   `json:"annotations"`
-	Ports       []types.ExposedPort `json:"ports"`
+	ACVersion      types.SemVer         `json:"acVersion"`
+	ACKind         types.ACKind         `json:"acKind"`
+	Apps           AppList              `json:"apps"`
+	Volumes        []types.Volume       `json:"volumes"`
+	Isolators      []types.Isolator     `json:"isolators"`
+	Annotations    types.Annotations    `json:"annotations"`
+	Ports          []types.ExposedPort  `json:"ports"`
+	CRIAnnotations types.CRIAnnotations `json:"criAnnotations,omitempty"`
```

Comment:
@squeed CRIAnnotation and CRILabels sounds too specific for CRI? Why not just using the old Annotations and changing the type of the Key from ACIdentifier to string?
Both Label or Annotation can be muxed into annotations with prefixes.